extensions/2.0/Khronos/KHR_physics_rigid_bodies/schema/glTF.KHR_physics_rigid_bodies.material.schema.json

```diff
@@ -3,18 +3,18 @@
     "$id": "glTF.KHR_physics_rigid_bodies.material.schema.json",
     "title": "KHR_physics_rigid_bodies Physics Material",
     "type": "object",
-    "description": "Parameters describing how an object should respond to collisions during physics simulation.",
+    "description": "Parameters describing the physical properties of a surface in a physics simulation.",
```

Comment:
You may disagree with me on this, but "how an object should respond" seems too vague to me. Whether an object is a trigger or not, and what the object's motion properties are, also determine "how an object should respond".

Reply:
Collision response is a technical term used in physics simulation. Obviously, these are not the _only_ parameters which determine the response, but I don't want to loose that specificity.